projects/storefrontapp-e2e-cypress/cypress/helpers/address-book.ts

```diff
@@ -100,30 +102,35 @@ export function addSecondAddress() {
     .contains(' Add new address ')
     .click({ force: true });
   fillShippingAddress(secondAddress);
-  cy.get('cx-address-card').should('have.length', 2);
+  cy.get('cx-card').should('have.length', 2);
 }
 
 export function setSecondAddressToDefault() {
-  cy.get('.set-default').click();
+  cy.get('a')
+    .contains('Set as default')
+    .click();
 
-  const firstCard = cy.get('cx-address-card').first();
+  const firstCard = cy.get('cx-card').first();
   firstCard.should('contain', '✓ DEFAULT');
   firstCard.should('contain', 'N Z');
 }
 
 export function deleteExistingAddress() {
-  let firstCard = cy.get('cx-address-card').first();
+  const firstCard = cy.get('cx-card').first();
 
-  firstCard.find('.delete').click();
-  cy.get('.cx-address-card-delete-msg').should(
+  firstCard
+    .get('a')
```

Comment:
`.get()` can only be chained off of `cy` (as seen here https://docs.cypress.io/api/commands/get.html#Requirements) 
Your code works, but is wrong - `.get()` looks for the element on the whole page, not from `firstCard`.

Reply:
You can try with `within` method and some fancy handler function (check this https://docs.cypress.io/api/commands/within.html#Get-inputs-within-a-form-and-submit-the-form)